Source/WebKit/WebProcess/GPU/media/WebMediaStrategy.cpp

```diff
@@ -80,4 +85,26 @@ std::unique_ptr<WebCore::NowPlayingManager> WebMediaStrategy::createNowPlayingMa
     return WebCore::MediaStrategy::createNowPlayingManager();
 }
 
+#if ENABLE(MEDIA_SOURCE)
+void WebMediaStrategy::enableMockMediaSource()
+{
+    m_useGPUProcessForMockMediaSource = true;
```

Comment:
Why are we setting it to true?
It seems then that, whenever we are using mock and GPUProcess, we will do mock in GPUProcess.
Except if after calling enableMockMediaSource(), we call setUseGPUProcessForMockMediaSource(false), which is somehow confusing.